### DISCUSSION TITLE
Upgrade failed going form 2.3.21to 2.3.30

### DISCUSSION THREAD

**Discussion by Wilk4013** [Security-Onion-Solutions/securityonion#3252] [Unsupported Versions] [answered]
I am trying to upgrade from 2.3.21 to 2.3.30 and I get the following error: 

There is a problem downloading the so-steno:2.3.30 image. Details:

gpg: Signature made Thu 18 Feb 2021 02:44:47 PM UTC using RSA key ID FE507013 gpg: BAD signature from "Security Onion Solutions, LLC <info@securityonionsolutions.com>"

**Comment by defensivedepth** [comment, marked as answer]
Try running soup again

**Reply by Wilk4013** [reply]
Josh,
I ran soup 2 more times and got the same. Here is the error:

"Command failed with exit code 1; will retry in 10 seconds (50 / 50)...
Command continues to fail; giving up.
Executing command with retry support: curl -A 'soup/2.3.21/centos/3.10.0-1160.11.1.el7.x86_64' https://sigs.securityonion.net/2.3.30/so-steno:2.3.30.sig --output /root/sosigs/so-steno:2.3.30.sig
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   543  100   543    0     0    213      0  0:00:02  0:00:02 --:--:--   213
Results:  (0)
There is a problem downloading the so-steno:2.3.30 image. Details:

gpg: Signature made Thu 18 Feb 2021 02:44:47 PM UTC using RSA key ID FE507013 gpg: BAD signature from "Security Onion Solutions, LLC <info@securityonionsolutions.com>"

**Reply by TOoSmOotH** [reply]
Try sudo so-docker-refresh on the manager

**Reply by Wilk4013** [reply]
Got a different error running so-docker-refresh:

so-docker-refresh
This is a manager, We can proceed.
Executing command with retry support: curl -sSL https://raw.githubusercontent.com/Security-Onion-Solutions/securityonion/master/KEYS -o /root/sosigs/KEYS
Results:  (0)
gpg: key FE507013: "Security Onion Solutions, LLC <info@securityonionsolutions.com>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1
Downloading so-acng
Executing command with retry support: docker pull ghcr.io/security-onion-solutions/so-acng:2.3.21
Results: 2.3.21: Pulling from security-onion-solutions/so-acng
Digest: sha256:bcc1cbc1042028c5b56ff68b9d578f1d8b5c7f29580385e092ded9f7fe6b8d28
Status: Image is up to date for ghcr.io/security-onion-solutions/so-acng:2.3.21
ghcr.io/security-onion-solutions/so-acng:2.3.21 (0)
Executing command with retry support: curl -A 'refresh/2.3.21/centos/3.10.0-1160.11.1.el7.x86_64' https://sigs.securityonion.net/2.3.21/so-acng:2.3.21.sig --output /root/sosigs/so-acng:2.3.21.sig
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   543  100   543    0     0   1144      0 --:--:-- --:--:-- --:--:--  1145
Results:  (0)
There is a problem downloading the so-acng:2.3.21 image. Details:

gpg: Signature made Mon 21 Dec 2020 10:52:15 PM UTC using RSA key ID FE507013 gpg: BAD signature from "Security Onion Solutions, LLC <info@securityonionsolutions.com>"

**Reply by TOoSmOotH** [reply]
Do you have any SSL intercept?

**Reply by lock-wire** [reply]
I had the same problem on the same container yesterday.  I rebooted the system (standalone install with features enabled on a VM) and re-ran soup.  The update completed with no further issues.

**Reply by Wilk4013** [reply]
TOoSmOotH,
I do have it on my firewall but I have an exception for the SO platform. I will look at that again.
lock-wire I will give that reboot a try also. Thanks

**Reply by Wilk4013** [reply]
Guys,
I was unable to resolve the issue with a reboot so I rebuilt my system from the .iso file. Thanks for all of your suggestions

**Reply by TOoSmOotH** [reply]
@zpravaiz please start your own thread. Your issue is different since it is trying to use content trust which is something we haven't used in a while.
